fuzzers/binary_only/qemu_launcher/src/client.rs

```diff
@@ -95,10 +94,14 @@ impl Client<'_> {
 
         let is_cmplog = self.options.is_cmplog_core(core_id);
 
-        let extra_tokens = injection_module
-            .as_ref()
-            .map(|h| h.tokens.clone())
-            .unwrap_or_default();
+        let extra_tokens = if cfg!(feature = "injections") {
+            injection_module
+                .as_ref()
+                .map(|h| h.tokens.clone())
+                .unwrap_or_default()
+        } else {
+            Vec::new()
```

Comment:
@rmalmain is there a better option for this? `.maybe_extra_token` that takes a `Some` or so?
Also, we should find a way to add CI for this